Formula/libtatsu.rb

```diff
@@ -0,0 +1,41 @@
+class Libtatsu < Formula
+  desc "Library handling the communication with Apple's Tatsu Signing Server (TSS)"
+  homepage "https://github.com/libimobiledevice/libtatsu"
+  url "https://github.com/libimobiledevice/libtatsu.git",
+    revision: "e3eb2e224781f6522e2cf63c35c2c249747d648d"
+  version "1.0.3"
```

Comment:
since tagged release version is used, please replace it with a url to a specific version – see fsmon for example
something like this:
```ruby
url "https://github.com/libimobiledevice/libtatsu/archive/refs/tags/1.0.3.tar.gz"
sha256 "3e039497bb90439dc01ead2b78ccbdf76dfce21b3fdd8f21a1ac584934e33a44"
```